runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingTracker.java

```diff
@@ -36,11 +36,13 @@
 import java.util.Set;
 import javax.annotation.Nullable;
 import javax.inject.Inject;
+
```

Comment:
stray whitespace

Reply:
addressed in 89158ef56cf9254654b0a8215cc0861f94350c49